docs/src/configuration/app/app-reference.md

```diff
@@ -187,6 +187,14 @@ web:
 This command runs every time your app is restarted, regardless of whether or not new code is deployed.
 So it can be useful for things like clearing ephemeral cache.
 
+```yaml {location=".platform.app.yaml"}
+web:
+    commands:
+        start: 'redis-cli -h redis.internal flushall; sleep infinity'
```

Comment:
What is the precise difference between the two?

Reply:
The one with `-h redis.internal` specifies the host name for Redis. It's different depending on the infrastructure.